Neos.Neos/Resources/Private/Fusion/Prototypes/ImageTag.fusion

```diff
@@ -0,0 +1,37 @@
+# ImageTag object works exactly the same way as image ViewHelper in the Neos.Media package
```

Comment:
i know its not your comment but that comment doesnt help much ... i dont know how the ViewHelper works and i dont want to :D